qdrant.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: qdrant
+  version: 1.6.1
+  epoch: 0
+  description: "High-performance, massive-scale Vector Database for the next generation of AI"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang-16-dev
+      - curl
+      - jq
+      - libgit2-dev
+      - llvm-cmake-16
+      - llvm-lld-16-dev
+      - protobuf-dev
+      - rust
+      - wget
+      - wolfi-base
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/qdrant/qdrant
+      tag: v${{package.version}}
+      expected-commit: d2d7328bb458d8e8439aa8d379f4f69493ee5198
+
+  - name: static-file
+
+  - runs: |
+      mkdir -p static
+      ./tools/sync-web-ui.sh
+
+  - name: build-qdrant-binary
+    runs: |
+      cargo build --release --bin qdrant
```

Comment:
I must admit I'm not 100% sure of the recommended way to build rust applications, I think I heard recently that using `cargo build ...` isn't the right way but let's try and confirm that.  

@kaniini do you have thoughts or another recommended way?

Reply:
i took a look at upstream workflow and they are using `taiki-e/upload-rust-binary-action` which use `cargo` by default https://github.com/taiki-e/upload-rust-binary-action/blob/v1/main.sh#L168